stable/drone/values.yaml

```diff
@@ -70,29 +59,33 @@ ingress:
 
 server:
   ## If not set, it will be autofilled with the cluster host.
-  ## Host must be in "<scheme>://<hostname>" format.
-  ##
-  # host: "https://drone.domain.io"
+  ## Host must be the public hostname of the server
+  # host: "drone.domain.io"
+
+  ## Scheme must match the public hostname, ie if ingress TLS or let's encrypt
+  ## are enabled, scheme must be https.
+  # scheme: "https"
+
+  ## Control the port the server listens on and whether or not cert/key is
+  ## generated for the server.
+  letsEncrypt: false
 
   ## Drone server configuration.
   ## Values in here get injected as environment variables.
-  ## ref: http://readme.drone.io/admin/installation-reference
+  ## ref: https://docs.drone.io/reference/server/
+  ## ref: https://docs.drone.io/reference/environ/
   ##
   env:
-    DRONE_DEBUG: "false"
+    DRONE_LOGS_DEBUG: "false"
     DRONE_DATABASE_DRIVER: "sqlite3"
     DRONE_DATABASE_DATASOURCE: "/var/lib/drone/drone.sqlite"
 
     ## Drone requires some environment variables to bootstrap the
     ## git service or it won't start up.
     ## Uncomment this and add your own custom configuration.
     ##
-    # DRONE_PROVIDER: "github"
-    # DRONE_OPEN: "true"
-    # DRONE_GITHUB: "true"
-    # DRONE_ORGS: "my-github-org,my-other-github-org"
```

Comment:
You're removing DRONE_ORGS but are not including its replacement, `DRONE_USER_FILTER`. It feels like that value is import and people should be using it for private installs and so including it would still be a good idea.